### PR TITLE
FFmpeg-Plugin: Fix memleak issues

### PR DESCRIPTION
**Pull request by jsunintel**

### CODE REVIEW

**Comment by jsunintel** [comment]
@lijing0010 @tianjunwork @hassount Please kindly review.

File: ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch [hunk 1]
```diff
@@ -407,8 +408,8 @@ index 0000000..d9ac04c
 +            header_ptr->nAllocLen = 0;
 +            header_ptr->nFilledLen = 0;
 +            header_ptr->nTickCount = 0;
-+            header_ptr->pBuffer = NULL;
 +            header_ptr->nFlags = EB_BUFFERFLAG_EOS;
++            header_ptr->pBuffer = NULL;
```
Comment (jsunintel) [inline ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch]: @tianjunwork Please check if GST has similar memleak.
SVT HEVC Lib asks us to set the input buffer's pBuffer to NULL, while we need to free that when closing.

Comment (tianjunwork) [reply ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch]: Sure, will do. Thanks for reminding.

File: ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch [hunk 2]
```diff
@@ -175,9 +176,10 @@ index 0000000..d9ac04c
 +
 +static void free_buffer(SvtContext *svt_enc)
 +{
-+    uint8_t *in_data = svt_enc->in_buf.pBuffer;
```
Comment (tianjunwork) [inline ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch]: svt_enc->in_buf.pBuffer points to in_data at line 194. And it is freed here. 
There should not be mem leak.

Comment (jsunintel) [reply ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch]: svt_enc->in_buf.pBuffer is set to NULL, when trying to send the last frame to SVT. And we lost the address for memory free.

Comment (tianjunwork) [reply ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch]: I see.

**Pull request merged**